test/www/jxcore/test/mockmobilegetdocpath.js

```diff
@@ -0,0 +1,17 @@
+var errArg;
+var fileLocationArg;
+
+function MobileGetDocumentsPath() {
+  return;
+}
+
+MobileGetDocumentsPath.storeGetDocumentsPathReturnArguments = function () {
+  errArg = arguments[0];
+  fileLocationArg = arguments[1];
+};
+
+MobileGetDocumentsPath.GetDocumentsPath = function (cb) {
+  cb(errArg, fileLocationArg);
+};
+
+global.Mobile = MobileGetDocumentsPath;
```

Comment:
Why do you kill the entire mobile object? Wouldn't it make much more sense to move this into the mockmobile file above and just take over storeGetDocumentsPathReturnArguments and GetDocumentsPath?